client/templates/course/lesson/quiz/questions/autoform/afQuestionOptionsGroup.js

```diff
@@ -0,0 +1,123 @@
+Template.afFieldQuestionOptionsGroup.helpers({
+
+    displayedOptions: function(){
+        var question = Session.get("currentQuestionToBuild");
+
+        var displayedOptions = [];
+        var allOptions = Template.instance().data.selectOptions;
+
+        var totalQuestions = question && question.optionTitles ? question.optionTitles.length : 0;
+
+        for (var i= 0; i< totalQuestions; i++){
+            displayedOptions.push (allOptions[i]);
+        }
+
+        return displayedOptions;
+    },
+
+    isHidden: function(){
```

Comment:
Again, overly complicated. The `isHidden` question property may not prove useful in the long term.